ironfish-cli/src/commands/accounts/pay.ts

```diff
@@ -70,16 +71,20 @@ export class Pay extends IronfishCommand {
 
     const status = await client.status()
 
-    if (!status.content.blockchain.synced) {
-      this.log(
-        `Your node must be synced with the Iron Fish network to send a transaction. Please try again later`,
-      )
-      this.exit(1)
-    }
+    // if (!status.content.blockchain.synced) {
+    //   this.log(
+    //     `Your node must be synced with the Iron Fish network to send a transaction. Please try again later`,
+    //   )
+    //   this.exit(1)
+    // }
 
     if (!amount || Number.isNaN(amount)) {
       const response = await client.getAccountBalance({ account: from })
 
+      const estimatedFee = await client.getEstimatedFee()
+
+      console.log("Estimated fee is: " + estimatedFee.content.fee);
```

Comment:
This is printed before asking the amount, not the fee. If you pass the `--amount` flag then this won't be printed.